docs/container-selection.md

```diff
@@ -7,29 +7,67 @@ There are two options:
 
 ## Full Exclude 
 
-If you need to exclude some containers, set the _com.centurylinklabs.watchtower.enable_ label to `false`.
-
-```docker
-LABEL com.centurylinklabs.watchtower.enable="false"
-```
-
-Or, it can be specified as part of the `docker run` command line:
-
-```bash
-docker run -d --label=com.centurylinklabs.watchtower.enable=false someimage
-```
-
-If you need to [include only containers with the enable label](https://containrrr.github.io/watchtower/arguments/#filter_by_enable_label), pass the `--label-enable` flag or the `WATCHTOWER_LABEL_ENABLE` environment variable on startup and set the _com.centurylinklabs.watchtower.enable_ label with a value of `true` for the containers you want to watch.
-
-```docker
-LABEL com.centurylinklabs.watchtower.enable="true"
-```
-
-Or, it can be specified as part of the `docker run` command line:
-
-```bash
-docker run -d --label=com.centurylinklabs.watchtower.enable=true someimage
-```
+If you need to exclude some containers, set the _com.centurylinklabs.watchtower.enable_ label to `false`.  For clarity this should be set **on the container(s)** you wish to be ignored, this is not set on watchtower.
+
+=== "dockerfile"
+
+    ```docker
+    LABEL com.centurylinklabs.watchtower.enable="false"
+    ```
+=== "dockerrun"
+
+    ```bash
+    docker run -d --label=com.centurylinklabs.watchtower.enable=false someimage
+    ```
+
+=== "docker-compose"
+
+    ``` yaml
+    version: "3"
+    services:
+      someimage:
+        command:
+          - serve
+          - --dev-addr=0.0.0.0:8000
+        container_name: someimage
+        entrypoint:
+          - somepoint
+        environment:
+          - PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+        labels:
+          - "com.centurylinklabs.watchtower.enable=false"
+    ```
+
+If instead you want to [only include containers with the enable label](https://containrrr.github.io/watchtower/arguments/#filter_by_enable_label), pass the `--label-enable` flag or the `WATCHTOWER_LABEL_ENABLE` environment variable on startup for watchtower and set the _com.centurylinklabs.watchtower.enable_ label with a value of `true` on the containers you want to watch.
+
+=== "dockerfile"
+
+    ```docker
+    LABEL com.centurylinklabs.watchtower.enable="true"
+    ```
+=== "dockerrun"
+
+    ```bash
+    docker run -d --label=com.centurylinklabs.watchtower.enable=true someimage
+    ```
+
+=== "docker-compose"
+
+    ``` yaml
+    version: "3"
+    services:
+      someimage:
+        command:
+          - serve
+          - --dev-addr=0.0.0.0:8000
+        container_name: someimage
+        entrypoint:
+          - somepoint
+        environment:
+          - PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment:
```suggestion
      someimage:
        container_name: someimage
```
I would suggest leaving out all the stuff that is not relevant here, especially since they have "real" values. Setting the `PATH` to `/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin` is not relevant to this feature, so it would be up to the user reading the docs to guess what matters and what doesn't.